composition-js/src/__tests__/compose.composeDirective.test.ts

```diff
@@ -998,4 +998,37 @@ describe('composing custom core directives', () => {
     const appliedDirectives = schema.elementByCoordinate('Query.shared')?.appliedDirectives;
     expect(appliedDirectives?.map(d => [d.name, d.arguments()])).toMatchObject([['auth', { scope: 'VIEWER'}], ['auth', {}]]);
   });
+
+  it('existing @authenticated directive with fed 1', () => {
```

Comment:
Very minor, but this file is for tests that use `@composeDirective`. I'd put it into `compose.test.ts` instead.

Reply:
my bad i was using this for testing and left it in a wrong spot 👍 will move it